.github/workflows/gcp-deploy.yaml

```
@@ -496,6 +496,7 @@ jobs:
             --set-env-vars OPENSEA_API_KEY=${{secrets.OPENSEA_API_KEY}} \
             --set-env-vars NEXT_PUBLIC_IMGIX_TOKEN=${{secrets.IMGIX_TOKEN}} \
             --set-env-vars NEXT_PUBLIC_GRAPHQL_URL=${{needs.env.outputs.GRAPHQL_URL}}
```

The newline needs to be escaped.

```suggestion
            --set-env-vars NEXT_PUBLIC_GRAPHQL_URL=${{needs.env.outputs.GRAPHQL_URL}} \
```